app/scripts/templates/confirm_account_unlock.mustache

```diff
@@ -0,0 +1,22 @@
+<div id="main-content">
+  <header>
+    <h1 id="fxa-confirm-account-unlock-header">{{#t}}Unlock email sent{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success">
+      {{#t}}Email resent{{/t}}
+    </div>
+
+    <div class="graphic graphic-mail">{{#t}}Email Sent{{/t}}</div>
+
+    <p class="verification-email-message">{{#t}}For security reasons your account was put on lockdown. To perform account-level activities, please re-verify your account by clicking the verification link sent to %(email)s{{/t}}</p>
+
+    <div class="links">
+      <a id="back" class="left" href="#">{{#t}}Back{{/t}}</a>
+      <a id="resend" class="right" href="#">{{#t}}Resend email{{/t}}</a>
```

Comment:
The wording here is inline with what's already shipping in the android client, which is good.  However note that we currently do *not* send email at the time the account is locked.  So the first time the user sees this page, we will not have actually sent them any email and they will have to click "resend email" to get out of this state.

The lockout is probably rare enough, and the thing-to-do obvious enough, that this won't be a problem in practice.  But we should consider if this can re-worded to reflect reality.